runtime/src/snapshot_utils.rs

```diff
@@ -1636,7 +1636,7 @@ pub fn bank_to_incremental_snapshot_archive(
 
     let temp_dir = tempfile::tempdir_in(snapshots_dir)?;
     let storages = {
-        let mut storages = bank.get_snapshot_storages();
+        let mut storages = bank.get_snapshot_storages(Some(full_snapshot_slot));
         filter_snapshot_storages_for_incremental_snapshot(&mut storages, full_snapshot_slot);
```

Comment:
The call to `filter_snapshot_storages_for_incremental_snapshot()` will be removed in the next PR.